tests/queries/0_stateless/02751_ip_types_aggregate_functions_states.sql.j2

```diff
@@ -0,0 +1,176 @@
+-- Tags: no-parallel, no-fasttest
+
+{# this test checks backward compatibility of aggregate functions States against IPv4, IPv6 types #}
+
+{% set ip4_generator = "select number::UInt32::IPv4 ip from numbers(999999999,50) order by ip" %}
+{% set ip6_generator = "SELECT toIPv6(IPv6NumToString(toFixedString(reinterpretAsFixedString(number)||reinterpretAsFixedString(number), 16))) AS ip FROM numbers(1010011101, 50) order by ip" %}
```

Comment:
this will not work well in debug build for `finalizeAggregation` because of this:
https://github.com/ClickHouse/ClickHouse/blob/master/base/base/sort.h#L119-L121
it was done intentionally to prevent testing relying on stable sort of identical keys.
I fixed this test here:
https://github.com/ClickHouse/ClickHouse/pull/50551/commits/7057e0e25fd55f8a9cb9708da223883aaa8fe902